recipe/gen_patch_json.py

```diff
@@ -2255,6 +2255,38 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             )
             _replace_pin("pandas >=1.1", "pandas >=1.3", record["depends"], record)
 
+        # fix numba / numpy compatibility, see
+        # https://github.com/conda-forge/numba-feedstock/issues/90
+        # based on
+        # https://numba.readthedocs.io/en/stable/user/installing.html#version-support-information
+        numba_numpy_depends = {
+            # format: numba_version: numpy lower bound (">="), numpy upper bound ("<")
+            "0.57": ("1.19", "1.24"),
+            "0.56.4": ("1.18", "1.24"),
+            "0.56.3": ("1.18", "1.24"),
+            "0.56.2": ("1.18", "1.24"),
+            "0.56.0": ("1.18", "1.23"),
+            "0.55.2": ("1.18", "1.23"),
+            "0.55.0": ("1.18", "1.22"),
+            "0.55.1": ("1.18", "1.22"),
+            "0.54": ("1.17", "1.21"),
+            "0.53": ("1.15", "1.21"),
+            "0.52": ("1.15", "1.20"),
+            "0.51": ("1.15", "1.19"),
+            "0.50": ("1.15", "1.19"),
+            "0.49": ("1.15", "1.18"),
+            "0.48": ("1.15", "1.18"),
+            "0.47": ("1.15", "1.18"),
+        }
+
+        if record_name == "numba" and record.get("timestamp", 0) <= 1671537177000:
+            for numba_version, (_, numpy_ub) in numba_numpy_depends.items():
+                if pkg_resources.parse_version(numba_version) == pkg_resources.parse_version(
```

Comment:
Does this syntax allow for things like `0.57.1` being the same as `0.57.2` since the numba docs specify `0.57.x`?

Reply:
No, it doesn't. Thanks for pointing this out. Fixed in 4a355723.